kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ClusterServiceDependentResource.java

```diff
@@ -72,28 +75,51 @@ public Map<String, Service> desiredResources(
                 .flatMap(ProxyNetworkingModel.ClusterNetworkingModel::requiredSniLoadbalancerPorts)
                 .distinct().sorted().toList();
 
-        var sniServiceStream = sniLoadbalancerServices(primary, sharedSniLoadbalancerPorts);
+        Set<BootstrapServersAnnotation.BootstrapServer> bootstraps = getLoadBalancerServiceBootstrapServers(clusterNetworkingModels);
+
+        var sniServiceStream = sniLoadbalancerServices(primary, sharedSniLoadbalancerPorts, bootstraps);
 
         return Stream.concat(serviceStream, sniServiceStream).collect(toByNameMap());
     }
 
-    private ObjectMeta serviceMetadata(KafkaProxy primary, String name) {
-        return new ObjectMetaBuilder()
+    /**
+     * Get the bootstrap servers hosted by the shared LoadBalancer Service
+     */
+    @NonNull
```

Comment:
can remove these NonNulls as it's the default annotation, IDEA likes to add these when I extract method